src/nupic/algorithms/ColumnPooler.cpp

```diff
@@ -54,7 +54,12 @@ using namespace nupic::algorithms::connections;
 using nupic::algorithms::temporal_memory::TemporalMemory;
 
 
-// TODO: Connections learning rules are different.
+// TODO: Connections learning rules are different...
+// Don't apply the same learning update to a synapse twice in a row, because
+// otherwise staring at the same object for too long will mess up the synapses.
+// This change allows it to work with timeseries data which moves very slowly,
+// instead of the usual HTM inputs which reliably change every cycle. See kropff
```

Comment:
:+1: this is a great addition!

Reply:
eventually, I think this should be addressed with a more realistic synapse model - as in Hodgkin-Huxley: potential decay, inhibition in short period after fireing,...